AirCasting/SessionsSynchronization/Controller/Utils/ScheduledSessionSynchronizerProxy.swift

```diff
@@ -9,6 +9,36 @@ import Combine
 // be the case tho)
 import class UIKit.UIApplication
 import struct UIKit.UIBackgroundTaskIdentifier
+import Resolver
+
+final class WiFiAwareSessionSynchronizerProxy: SessionSynchronizer {
```

Comment:
this deserves its own file. Please also take a look how the other proxy is documented and maybe try to do it as well